src/foam/nanos/menu/MenuDefinedViewList.js

```diff
@@ -0,0 +1,35 @@
+foam.CLASS({
+  package: 'foam.nanos.menu',
+  name: 'MenuDefinedViewList',
+  extends: 'foam.u2.View',
+
+  properties: [
+    {
+      name: 'containerView',
+      class: 'foam.u2.ViewSpec'
+    },
+    {
+      name: 'childViews',
+      class: 'FObjectArray',
+      of: 'foam.u2.View'
+    }
+  ],
+
+  methods: [
+    function initE() {
+      var self = this;
+      var view = this;
+
+      view = view
+        .start(self.containerView);
+      
+      for ( var i = 0; i < self.childViews.length; i++ ) {
+        view = view
+          .start(self.childViews[i]).end();
```

Comment:
When you have `.start(foo).end()` you can just do `.tag(foo)` instead. You only need start and end when you want to do other calls in between.